### PR TITLE
Building XML webpages using XSLT

### LINKED ISSUE #436: Building XML webpages using XSLT
### Brief Summary: 

XSLT (eXtensible Stylesheet Language Transformations) is a language for transforming XML documents into other XML documents, or other formats such as HTML for web pages, plain text or XSL Formatting Objects, which may subsequently be converted to other formats, such as PDF, PostScript and PNG. In this article, we will cover how XML content can be rendered as webpages by transformation using XSLT

### Key Takeaways: 

- The reader will learn about XSLT concepts
- The reader will understand how XML can be converted to webpages
- The reader will get good understanding by implementing XSLT transformations

### References: 

- https://www.w3.org/TR/xslt/

### PR DESCRIPTION
**Pull request by srishilesh**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #436

### CODE REVIEW

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

File: articles/building-xml-webpages-using-xslt/index.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+```
+
+To try out this, replace the above code snippet with `<!--ADDITIONAL FEATURE-->` under [Full Code - Table Rows](#full-code-table-rows).
+
+#### View Output
+By following the above step-by-step procedures, you would have successfully built a webpage. To view the page, you must open the file with a `.xml` extension.
+
+Due to security concerns in several browsers like Google Chrome, the output can be viewed using the Internet Explorer browser. More details about the security concern in Google Chrome can be found [here](https://stackoverflow.com/questions/4558160/xsl-not-working-in-google-chrome).
+
+![A sample output of XML Webpage](/engineering-education/building-xml-webpages-using-xslt/output.PNG)
```
Comment (lalith1403) [inline articles/building-xml-webpages-using-xslt/index.md:280]: Even on internet explorer, I fail to get the output. I see the code itself. Any alternatives?

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Note the few changes made on the grammatical side. Technically, I was not able to see the output. Clarify the same and this should be good to go.

File: articles/building-xml-webpages-using-xslt/index.md (hunk 1 above)
Comment (srishilesh) [reply articles/building-xml-webpages-using-xslt/index.md:280]: Hi @lalith1403. I had made a minor error in my XML document, so it didn't work. The fixed code can be found in the latest commit.

For reference, I have put links containing all the codes under "Conclusion". Please let me know if it doesn't work. Thanks.

**Comment by hectorkambow** [comment]
Plagiarism check done

File: articles/building-xml-webpages-using-xslt/index.md (hunk 1 above)
Comment (lalith1403) [reply articles/building-xml-webpages-using-xslt/index.md:280]: Yep. They worked. Thanks.

**Pull request merged**